core/src/test/java/io/github/tcdl/support/TestUtils.java

```diff
@@ -145,5 +150,13 @@ public static Payload createSimpleResponsePayload() {
     public static MetaMessageBuilder createSimpleMetaBuilder(MsbConfigurations msbConf, Clock clock) {
         return new MetaMessage.MetaMessageBuilder(null, clock.instant(), msbConf.getServiceDetails(), clock);
     }
+    
+    private static class TestMsbContext extends MsbContext {
```

Comment:
Now MsbContext has only protected Constructor. And new instance can be created only by means of MsbContext.Bulder (as a usual way) or with extended class (for test purposes).